src/middleware/session.ml

```diff
@@ -329,11 +329,15 @@ let {middleware; getter} =
 let two_weeks =
   60. *. 60. *. 24. *. 7. *. 2.
 
-let memory_sessions ?(lifetime = two_weeks) =
-  middleware (Memory.back_end lifetime)
+module Make (Pclock : Mirage_clock.PCLOCK) = struct
+  let now () = Ptime.to_float_s (Ptime.v (Pclock.now_d_ps ()))
 
-let cookie_sessions ?(lifetime = two_weeks) =
-  middleware (Cookie.back_end lifetime)
+  let memory_sessions ?(lifetime = two_weeks) =
```

Comment:
Could you do the same un-indentation here, and add a new commit with it, while I review the logger? Even though this part is small and I can review it even with the indentation mixed in, it's still better for blame to do the indentation of this chunk separately. When someone is checking the blame, they will click right past a commit that says "Adjust indentation after recent commits." Otherwise, they will face this diff and have to spend time realizing that the change here is actually much smaller than it looks in the diff, and potentially didn't even affect the parts that they are interested in. It was just indentation, but not immediately obvious.

Reply:
> It was just indentation, but not immediately obvious.

Referring to the function headers with this statement.